snapcraft.yaml

```diff
@@ -18,7 +18,8 @@ parts:
       - wget
     build-environment:
       - RELEASE: "24.04"
-      - BASE: ubuntu-base-${RELEASE}-base-${CRAFT_ARCH_BUILD_FOR}.tar.gz
+      - DOT_RELASE: "1"
```

Comment:
Mmm... `dot_relEase`... shouldn't it?

Reply:
(and the same in the next line, of course)

Reply:
Fixed, good catch!